README.md

```diff
@@ -91,9 +92,10 @@ We are maintaining different pre-defined configurations in "config" files.
     creator-platform-all.config - Basic Creator Platform profile enabling all the userspace and kernel space packages suitable for CC2520 based platforms.
     creator-platform-all-cascoda.config - Basic Creator Platform profile enabling all the userspace and kernel space packages suitable for Cascoda CA8210 based platforms.
 
-## To build for cascoda CA8210 platform:
+## To build for Cascoda CA8210 platform:
+
+_Note :- By default, CreatorKit project1 for Cascoda CA8210 platform is built if no config is specific using P= option._
```

Comment:
may be good to rephrase to:

By default, CreatorKit project1 is built for Cascoda CA8210 platform, if no config has been specified using P= option